docs/index.rst

```diff
@@ -73,31 +75,27 @@ the ``postgresql://`` JDBC driver prefix, the `Apache Flink JDBC Connector`_
 will implicitly select the corresponding dialect implementation for PostgreSQL.
 
 In turn, this will employ a few behaviours that strictly expect a PostgreSQL
-server on the other end, so that some operations will croak on databases
+server on the other end, so that some operations will fail on databases
 offering wire-compatibility with PostgreSQL, but do not provide certain
 features like the `hstore`_ or `jsonb`_ extensions. Also, tools like `Dataiku`_
 need this driver to implement transaction commands like ``ROLLBACK`` as a
 no-op.
 
 
-.. _implementations:
-.. _jdbc-implementation:
+.. _differences:
 
-What's inside
-=============
+Differences
+===========
+
+The driver is based upon a fork of the `PostgreSQL JDBC Driver`_, see `pgjdbc
+driver fork`_. On a high-level perspective, this list enumerates a few
+behavioral differences.
 
-The driver is based upon a fork of the `PostgreSQL JDBC Driver`_, see
-`pgjdbc driver fork`_.
+- The CrateDB driver deserializes objects to a Map, the official one treats them as JSON.
+- A few metadata functions have been adjusted to better support CrateDB's type system.
```

Comment:
Thank you for your suggestions. With c61ebe2ff62, I've moved the internal details to a dedicated section, and started on the landing page with a list of differences, as you suggested.

However, I don't know much about this either, so I am humbly asking to review this list, and either guide me what should go in there, or support me with a few statements which could be added here.

Maybe @seut or @BaurzhanSakhariev could also share some thoughts or suggestions on this topic?

---
_The current state of this section can be inspected at [CrateDB legacy JDBC driver » Driver differences](https://github.com/crate/crate-jdbc/blob/amo/docs-iterate/docs/index.rst#differences)._